README.md

```diff
@@ -95,3 +99,12 @@ and [Sprig](https://github.com/Masterminds/sprig) library.
 ### Built-in functions
 There are a few built in functions as well:
   * `require (env "ENV_NAME")` - Renders an error if environments variable does not exists. If it is equal to empty string, returns empty string.  `{{ require (env "TIMEOUT_MS) }}`
+
+### Nested GO templates
```

Comment:
I dont like the uppercase `GO` -> `Go`

Reply:
https://github.com/seznam/goenvtemplator/pull/14/commits/ad360b91d58314104e6512d8d6b70b159ab790ad